openapi/components/schemas/Applications/ApplicationInstanceConfiguration.yaml

```
@@ -0,0 +1,37 @@
+type: object
+required:
+  - settings
+description: |-
+  Application instance configuration.
+
+  Only user can change settings when `configuredBy` is `user`.
+
+  Only the application itself can change settings when `configuredBy` is `application`.
+properties:
+  settings:
+    description: Settings that are available for the user to complete when they install the application.
+    type: object
+    additionalProperties:
+      type: string
+    example:
+      color: red
+      level: gold
+  _links:
+    readOnly: true
+    type: array
```

nitpick: we need a description here 
```suggestion
    type: array
    description: Related links.
```